RELEASES.md

```diff
@@ -2,6 +2,10 @@
 
```

Comment:
Give me ~30 minutes, I'll rewrite that branch so we can ff-only merge it such that it makes sense in subnet-evm's history.